hw/dv/tools/dvsim/xcelium.hjson

```diff
@@ -35,6 +35,10 @@
                "-nowarn DSEMEL",
                // Ignore hierarchial ref warnings in interfaces
                "-nowarn CUVIHR",
+               // Needed for including "secded_enc.h".
+               "-I{build_dir}/src/lowrisc_dv_secded_enc_0",
+               // Needed to support "$assertcontrol" used in hw/top_earlgrey/dv/env/ast_supply_if.sv.
```

Comment:
Its not clear what Xcelium limitation does this solve. It may be worth adding more explanation to this.